src/components/PhonicWordSelector/PhonicWordSelector.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from "react";
+import { Button } from "reactstrap";
+import "../WordSelector/WordSelector.scss";
+import "../PhonicIcon/PhonicIcon.scss";
+function PhonicWordSelector(props) {
+  const [selected, setSelected] = useState(false);
+  function handleSelect() {
+    //if previously selected, unselect it
+    if (selected) {
+      props.handleSelectPhonics(false, props.name, props.index);
+    } else {
+      props.handleSelectPhonics(true, props.name, props.index);
+    }
```

Comment:
you can prob just do `props.handleSelectPhonics(!selected, props.name, props.index);` instead of the if-else